package.json

```diff
@@ -60,8 +59,6 @@
     "gzip-size": "^3.0.0",
     "inquirer": "^0.11.0",
     "jsdoc": "^3.4.0",
-    "jshint": "^2.9.1",
```

Comment:
This can't be removed. Please see https://github.com/ckeditor/ckeditor5-dev-utils/issues/5#issuecomment-235074578